driver/src/price_finding/optimization_price_finder.rs

```diff
@@ -281,16 +281,13 @@ impl Io for DefaultIo {
         &self,
         input_file: &str,
         result_folder: &str,
-        solver_type: SolverType,
+        solver: SolverType,
         time_limit: Duration,
     ) -> Result<()> {
         let time_limit = (time_limit.as_secs_f64().round() as u64).to_string();
         let output = Command::new("python")
-            .args(&["-m", "batchauctions.scripts.e2e._run"])
-            .arg(input_file)
-            .args(&["--outputDir", result_folder])
-            .args(&["--solverTimeLimit", &time_limit])
-            .args(solver_type.to_args())
+            .current_dir(solver.dir())
```

Comment:
Is there any advantage of making the caller have to change to the directory before? 

It introduces some implicit coupling between the methods (current_dir needs to be invoked together with args). If there is, I'd prefer if the solver could return the entire Command argument but keep that logic internal to the solver type.

Reply:
We have to change the dictionary, as the command:
```
python -m open_solver/src/match
```
would give me the error message that the module open_solver does not exist. It only works for batchauctions dir, as we use the virtual env in the image

Reply:
Why don't we have virtual env for the open solver?

Reply:
The switch in the Dir has the additional advantage that we don't have to import our python modules with complicated try statements as we currently do in the dex-solver: https://github.com/gnosis/dex-solver/blob/master/scripts/e2e/B_convert_input_int2float.py#L27

The virtual env would not solve this.

Reply:
Ok, what do you think about instead of exposing current_dir implement std::convert::Into<Command> on solver type so that this can stay an implementation detail of the solving strategy? Not a big deal if not but I think it would be cleaner to not make the caller deal with the ordering of the calls.